reference/platforms.md

```diff
@@ -54,4 +52,22 @@ Instead of supplying the display settings via userdata through the `amc launch`
 
 ### `webrtc` platform
 
-When using the [Stream Gateway](https://discourse.ubuntu.com/t/streaming-android-applications/17769), the `webrtc` platform is automatically used when launching containers. You don't need to perform additional steps. Launching a container with the webrtc platform can be done via the [web dashboard](https://discourse.ubuntu.com/t/web-dashboard/20871).
+The `webrtc` platform is the default platform used by Anbox to provide graphical output. It supports all GPUs supported by Anbox Cloud next to software rendering. It is used by default when a container is launched with enabled graphics (e.g. `amc launch --enable-graphics ...` or via the Anbox Stream Gateway)
```

Comment:
I would be careful with the word `default` as it could be understood as `webrtc is the default platform.`
Maybe something like
```suggestion
The `webrtc` platform is used by Anbox to provide graphical output. It supports all GPUs supported by Anbox Cloud next to software rendering. It is used when a container is launched with `--enabled-graphics`, or via the Anbox Stream Gateway.
```

Reply:
I agree, let's use the suggestion.